source/objects/responseClass.m

```diff
@@ -321,23 +321,33 @@
             end
         end
         
-        function obj = loadMoorDyn(obj,linesNum)            
+        function obj = loadMoorDyn(obj,linesNum, InputFile)            
```

Comment:
Use "inputFile" with camel case to be consistent. Same with "inputStrings" and "fileRootPath" below.

Reply:
Done, thanks for the review!